Mathlib/MeasureTheory/Function/AEMeasurableSequence.lean

```diff
@@ -9,19 +9,20 @@ Authors: Rémy Degenne
 ! if you have ported upstream changes.
 -/
 import Mathlib.MeasureTheory.MeasurableSpace
+import Mathlib.MeasureTheory.Measure.MeasureSpaceDef
```

Comment:
That import is new? Why is it needed?

Reply:
In mathlib3 this file imports `measure_theory.measurable_space` which imports the **tactic file** `measure_theory.tactic` which imports `measure_theory.measurable_space_def` which contains the declaration `measure_theory.measure` required for _this_ file. In mathlib4 there is no `MeasureTheory.Tactic` file, so the existing import chain as directly transported to mathlib4 doesn't have `MeasureTheory.Measure` available. So the extra import is needed.

Reply:
Thanks for the explanation!